openrlhf/trainer/ray/ppo_actor.py

```diff
@@ -349,6 +349,29 @@ def prepare_datasets(self):
             self.prompts_dataset, args.rollout_batch_size // strategy.world_size, True, True
         )
 
+        if args.eval_data:
+            eval_data = blending_datasets(
+                args.eval_data,
+                args.eval_data_probs,
+                strategy,
+                args.seed,
+                return_eval=False,
+            )
+
+            if args.limit_val_batches is not None:
+                samples_per_batch = args.rollout_batch_size // strategy.world_size
```

Comment:
Should this value be changed due to eval ? Wont it mess up training step too ?

Reply:
Which value? `samples_per_batch`? It is only used here. And mess up how?